test/spec/ol/renderer/canvas/canvasvectorlayerrenderer.test.js

```diff
@@ -5,14 +5,10 @@ describe('ol.renderer.canvas.VectorLayer', function() {
   describe('constructor', function() {
 
     it('creates a new instance', function() {
-      var map = new ol.Map({
-        target: document.createElement('div')
-      });
       var layer = new ol.layer.Vector({
         source: new ol.source.Vector()
       });
-      var renderer = new ol.renderer.canvas.VectorLayer(map.getRenderer(),
-          layer);
```

Comment:
I thought the same but didn't know what to do initially. Now I changed the tests so that this would be caught.